### PR TITLE
[Rust][CI] Restore Rust CI

### PR DESCRIPTION
**Pull request by jroesch**
Turn the Rust CI tests back on.

### CODE REVIEW

**Comment by jroesch** [comment]
Blocked on #5138

**Comment by jroesch** [comment]
cc @tqchen @nhynes

**Comment by tqchen** [comment]
Will report once master CI has been updated(then we need to rebase to retrigger this PR)

File: rust/runtime/tests/build_model.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def main():
     dshape = (32, 16)
     net = _get_model(dshape)
     mod, params = testing.create_workload(net)
+    import pdb; pdb.set_trace()
```
Comment (tqchen) [inline rust/runtime/tests/build_model.py:44]: remove this line, seems pdb cases segfault on the CI

**Comment by tqchen** [comment]
Thanks @jroesch ! cc @nhynes

**Pull request merged**